Umbraco 15.4 release documentation updates (#6921)
umbraco/UmbracoDocs: 15/umbraco-cms/reference/notifications/userservice-notifications.md

---
description: Example of how to use a UserService Notification
---

# UserService Notifications

The UserService implements `IUserService` and provides access to operations involving `IUser`.

> The following example illustrates how the password reset feature can be cancelled for a given set of users.

Reviewer: ```suggestion
The following example illustrates how to cancel the password reset feature for a given set of users.
```
